frontend/src/constants/notificationTypes.js

```diff
@@ -32,7 +32,7 @@ const NOTIFICATION_TYPES = {
   COMPLIANCE_REPORT_RECOMMENDED_FOR_REJECTION_MANAGER: 'Compliance Report Recommended for Rejection - Manager',
   COMPLIANCE_REPORT_ACCEPTED: 'Compliance Report Accepted',
   COMPLIANCE_REPORT_REJECTED: 'Compliance Report Rejected',
-  COMPLIANCE_REPORT_REQUESTED_SUPPLEMENTAL: 'Compliance Report Requested Supplemental',
+  COMPLIANCE_REPORT_REQUESTED_SUPPLEMENTAL: 'Compliance Report Supplemental Requested',
```

Comment:
This wording seems awkward to me. intentional?

Reply:
Ya, it is intentional. I just copied the suggestion from trello.